guardrails/cli/hub/uninstall.py

```diff
@@ -82,6 +61,9 @@ def uninstall(
     ),
 ):
     """Uninstall a validator from the Hub."""
+    from guardrails.hub.validator_package_service import ValidatorPackageService
+
+    print(f"ValidatorPackageService: {ValidatorPackageService}")
```

Comment:
still need this? should be a console.print or logger. ?

Reply:
good catch 👌🏼  shouldn't be there